ledger/accountdb.go

```diff
@@ -2097,6 +2114,50 @@ func performTxTailTableMigration(ctx context.Context, tx *sql.Tx, blockDb db.Acc
 	return err
 }
 
+func performOnlineRoundParamsTailMigration(ctx context.Context, tx *sql.Tx, blockDb db.Accessor, newDatabase bool, initProto protocol.ConsensusVersion) (err error) {
+	totals, err := accountsTotals(tx, false)
+	if err != nil {
+		return err
+	}
+	if newDatabase {
+		onlineRoundParams := []ledgercore.OnlineRoundParamsData{
+			{
+				OnlineSupply:    totals.Online.Money.Raw,
+				RewardsLevel:    totals.RewardsLevel,
+				CurrentProtocol: initProto,
+			},
```

Comment:
It seems like you could shorten the newDatabase code here and combine it with the logic below so it is just deciding whether to assign CurrentProtocol = initProto vs CurrentProtocol = hdr.CurrentProtocol